plugins/woocommerce/tests/e2e-pw/tests/merchant/create-restricted-coupons.spec.js

```diff
@@ -214,7 +224,9 @@ test.describe( 'Restricted coupon management', { tag: '@services' }, () => {
 						.first()
 						.pressSequentially( product.name );
 					await page
-						.getByRole( 'option', { name: product.name } )
+						.getByRole( 'option' )
+						.first()
+						.filter( { hasText: product.name } )
```

Comment:
Choose any product tat contains `product.name`, this will cover Pressable where can be more that one product with name containing `product.name`.

Reply:
I'm not sure about this. I see now that the product's name is `Product`. We should update this test to use unique product names. Updating the fixture to use `Product Date.now()` should be enough. We updated a lot of specs to use unique test data, this one was missed.

Reply:
I agree, it should be unique 👍 Done in [ccd5161](https://github.com/woocommerce/woocommerce/pull/51016/commits/ccd51617585363f8aabe4b6f2ff485d58f324b42)